app/values/pulfalight/location_code.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Pulfalight
+  class LocationCode
```

Comment:
Please see https://github.com/PrincetonUniversityLibrary/pulfa/blob/production/webapp/xsl/lib.xsl#L52 for where these values were originally derived.